tools/csv-merger/csv-merger.go

```diff
@@ -456,6 +499,20 @@ func main() {
 
 }
 
+func overwriteWithStandardLabels(labels map[string]string, version string, component hcoutil.AppComponent) {
+	appendedLabels := map[string]string{
```

Comment:
What's the use of the extra map + for loop? (except that it indents a bit more nicely)
Why not just this 4-liner:
```go
labels[hcoutil.AppManagedByLabel] = "olm"
labels[hcoutil.AppVersionLabel] = version
labels[hcoutil.AppPartOfLabel] = hcoutil.HyperConvergedCluster
labels[hcoutil.AppComponentLabel] = string(component)
```

Reply:
Good catch!